Backend/src/usecase/implementation/GameB2DConverter.ts

```diff
@@ -3,6 +3,10 @@ import { BoundaryGame } from "../model/BoundaryGame";
 
 export class GameB2DConverter {
     convert(boundary: BoundaryGame, word: string): Game {
+        return this.createGame(boundary, word);
+    }
+
+    private createGame(boundary: BoundaryGame, word: string): Game{
```

Comment:
labai nereikalingas šitas wrapper metodas. Viršuj public metodas TIK pakviečia šitą metodą, tai ar tikrai reikia pridėti complexity iškeliant kodą į naują metodą?